crates/redstone/src/contract/verification.rs

```diff
@@ -0,0 +1,168 @@
+//! Module containing primitives used for verification of the contract data and logic.
+//!
+//! See [verify_untrusted_update], [verify_trusted_update] and [verify_signers_config].
+
+use crate::{
+    network::error::Error, utils::slice::check_no_duplicates, SignerAddress, TimestampMillis,
+};
+
+/// MIN_TIME_BETWEEN_UPDATES_FOR_TRUSTED is set to 0,
+/// since trusted can update as long as write timestamp is increasing.
+const MIN_TIME_BETWEEN_UPDATES_FOR_TRUSTED: TimestampMillis = TimestampMillis::from_millis(0);
+/// MAX_SIGNER_COUNT describes maximum number of signers in Config.
+const MAX_SIGNER_COUNT: usize = u8::MAX as usize;
+
+/// Verify if between writes at least `min_time_between_updates` time passed.
```

Comment:
Can we document the function in the same manner like `verify_signers_config` from line 97 to make it consistent? WDYT?

Reply:
Sure, please check if now its fine (I will resolve rest of the comments and leave this one unresolved)

Reply:
Proceed!